### PR TITLE
Add days since last commit to options 🖐

### PR DESCRIPTION
**Pull request by olso**

### CODE REVIEW

File: webext/data/contentscript.js [hunk 1]
```diff
@@ -129,6 +131,12 @@ function clearLocalStorage() {
     }
 }
 
+function olderThanDays(date, days) {
```
Comment (olso) [inline webext/data/contentscript.js]: Doesn't handle DST, but whatevs. Didn't want to import `moment` just for this

Comment (musically-ut) [reply webext/data/contentscript.js]: Reasonable call. 👍

**Comment by musically-ut** [comment]
I've verified that this works for both Firefox and Chrome and don't have a problem adding this feature.

Are you satisfied with this PR or are you planning on adding something else? 

Otherwise, LGTM.

**Comment by olso** [comment]
@musically-ut Merge 👷

**Pull request merged**